js/script.js

```diff
@@ -0,0 +1,198 @@
+import mydata from '../mydata.js';
+
+console.log(mydata);
```

Comment:
- [ ] OPTIONAL: I recommend you delete this console.log statement, to clean up the repo.